inc/thorvg.h

```diff
@@ -1741,7 +1742,7 @@ class TVG_API WgCanvas final : public Canvas
      * @see Canvas::viewport()
      * @see Canvas::sync()
      */
-    Result target(void* instance, void* surface, uint32_t w, uint32_t h, void* device = nullptr) noexcept;
+    Result target(void* instance, void* surface, uint32_t w, uint32_t h, int type = 0, void* device = nullptr) noexcept;
```

Comment:
I'd like to suggest parameter order and names: (device, instance, target, w, h, type = 0)